RELEASE.md

```diff
@@ -0,0 +1,22 @@
+# Release process
+
+## Shipping a new version
+
+Once we have a build on master we want to publish:
+
+1. Run CardScan systems iOS test
```

Comment:
should we run `Carthage build` or something else just to make sure that the carthage part still works?